aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -71,6 +71,7 @@ module aptos_framework::account {
 
     struct SignerCapabilityOfferProofChallenge has drop {
         sequence_number: u64,
+        source_address: address,
```

Comment:
This will fail upgrade compatibility checks since an existing struct can't be modified.

Reply:
should we do something like `SignerCapabilityOfferProofChallengeV2`? a bit ugly, but should be compatible

Reply:
Yep, done!